plugin.py

```diff
@@ -58,11 +66,50 @@ def platform() -> str:
         return "unknown-linux-gnu"
 
 
+def open_runnables_in_terminus(window: Optional[sublime.Window], runnables: List[Runnable]) -> None:
+    filtered_runnables = [r for r in runnables if r["kind"] == "cargo"]
+    if len(filtered_runnables) == 0:
+        return
+    if not window:
+        return
+    view = window.active_view()
+    if not view:
+        return
+    if not Terminus:
+        sublime.error_message(
+            'Cannot run executable. You need to install the "Terminus" package and then restart Sublime Text')
+        return
+    main_cargo = shutil.which("cargo")
+    if not main_cargo:
+        sublime.error_message('Cannot find "cargo" on path.')
+        return
+    main_cargo_path = '"{}"'.format(main_cargo)
+    for runnable in filtered_runnables:
+        if runnable["args"]["overrideCargo"]:
```

Comment:
```suggestion
        args = runnables["args"]
        if args["overrideCargo"]:
```

This code does excessive access to `["args"]`. It'd be better to just assign it once and then work with it. Python doesn't optimise this kind of repeated access and it's more readable anyway.

Note that `args` near the end is reassigned into a mapping but this is done after we're done working with the `args` that represents the runnable arguments.

Reply:
for sure makes sense to clean up repetitive code like that